sdk-commons/src/main/java/org/openmrs/maven/plugins/model/DistroProperties.java

```diff
@@ -127,15 +127,12 @@ public Artifact getParentArtifact() {
 
         int missingCount = 0;
         if (StringUtils.isBlank(parentArtifactId)) {
-            log.warn("parent.artifactId  missing");
             missingCount++;
         }
         if (StringUtils.isBlank(parentGroupId)) {
-            log.warn("parent.groupId is missing");
             missingCount++;
         }
         if (StringUtils.isBlank(parentVersion)) {
-            log.warn("parent.version is missing");
```

Comment:
Not sure why we're removing these lines?

Reply:
@ibacher / @wikumChamith ... Yeah, not related to this PR, but I removed them because they seem incorrect and I noticed them in my log and investigated.  If I have no parent properties at all in my distro, I get these 3 warning messages in my console, which is wrong.  We can certainly keep these warnings in place if we first check that at least one property is entered, but honestly I don't understand the logic, because the conditional statement below seems to render all this moot?  Based on the conditional below, if you have 1 or 2 missing, then things fail.  So the only allowed situation is 0 properties or all 3 properties.  So why are we logging a warning above exactly?